ts/features/bonus/bpd/saga/networking/paymentMethod.ts

```diff
@@ -58,18 +60,23 @@ export function* bpdLoadPaymentMethodActivationSaga(
     );
     if (findPaymentMethodResult.isRight()) {
       if (findPaymentMethodResult.value.status === 200) {
-        yield put(
+        return yield put(
```

Comment:
```suggestion
       const payload = convertNetworkPayload(findPaymentMethodResult.value.value);
       yield put(bpdPaymentMethodActivation.success(payload))
       return payload;
```

`put` returns a `PutEffect`
Do you what return a `PutEffect` or the payload dispatched within `bpdPaymentMethodActivation.success` action?

Reply:
restored in e13b155